stripper/ze_m0w0m_csgo1.cfg

```diff
@@ -0,0 +1,12 @@
+;A missing TP Angle fix
```

Comment:
Are you sure this isn't intended? It would be very odd for only a single trigger_teleport to be accidentally missing UseLandmarkAngles.

Reply:
in old mowom csgo version, it did not have any "UseLandmarkAngles" "1". In mowom csgo1, i guess the porter forgot about that Teleport.